coral-spark/src/main/java/com/linkedin/coral/spark/CoralSpark.java

```diff
@@ -241,4 +266,20 @@ public List<SparkUDFInfo> getSparkUDFInfoList() {
   public String getSparkSql() {
     return sparkSql;
   }
+
+  /**
+   * Getter for the SqlNode representation
+   *
+   * @return SqlNode
+   */
+  public SqlNode getSqlNode() {
+    return sqlNode;
+  }
+
+  /**
+   * Setter for the Spark SQL
+   */
+  public void setSparkSql(String sparkSql) {
+    this.sparkSql = sparkSql;
```

Comment:
Passing Spark SQL in constructor as well as setter does not sound right. What is the reasoning? Can we avoid it?

Reply:
we dont need the setter with Spark-View-Catalog. Removed it